core/services/ardupilot_manager/frontend/index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>ArduPilot Manager</title>
+    <link rel="stylesheet" href="/static/css/metro-all.css" />
+    <link rel="stylesheet" href="/style.css" />
+    <script src="/static/js/vue.js"></script>
+    <script src="/static/js/axios.min.js"></script>
+    <script src="/static/js/metro.js"></script>
+</head>
+
+<body>
+    <div id="app">
+        <div class="container">
+            <h1>ArduPilot Manager</h1>
+            <h3>Endpoints:</h3>
+
+            <div class="grid">
+                <div class="row p-1 m-1 bg-gray">
+                    <div class="cell-2">
+                        <div>Name</div>
+                    </div>
+                    <div class="cell-1">
+                        <div>Owner</div>
+                    </div>
+                    <div class="cell-2">
+                        <div class="d-flex flex-justify-end">Type</div>
+                    </div>
+                    <div class="cell-2">
+                        <div class="d-flex flex-justify-center">Place</div>
+                    </div>
+                    <div class="cell-2">
+                        <div class="d-flex flex-justify-start">Argument</div>
+                    </div>
+                    <div class="cell-1">
+                        <span class="mif-floppy-disk mif-lg d-flex flex-justify-center pr-2"></span>
+                    </div>
+                    <div class="cell-1">
+                        <span class="mif-lock mif-lg d-flex flex-justify-center pr-2"></span>
```

Comment:
It would be nice to have a tooltip for this icons since it's not totally clear the idea behind each one.

Reply:
Good idea! Will add.

Reply:
Added.
It's not pretty currently, [because of a MetroUI bug](https://github.com/olton/Metro-UI-CSS/issues/1774).